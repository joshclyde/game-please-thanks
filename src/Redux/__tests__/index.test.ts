import { createStore } from "redux";

import { reducers } from "../";

describe(`GIVEN a redux store with the reducers`, () => {
  let store: ReturnType<typeof createStore>;
  beforeEach(() => {
    store = createStore(reducers);
  });
  test(`THEN the initial state should be as expected`, () => {
    expect(store.getState()).toMatchSnapshot();
  });
});
