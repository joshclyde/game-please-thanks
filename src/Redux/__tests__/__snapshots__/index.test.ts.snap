// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN a redux store with the reducers THEN the initial state should be as expected 1`] = `
Object {
  "auth": Object {
    "isAuthenticated": false,
  },
  "design": Object {
    "form": Object {},
    "iconTabs": Object {},
  },
  "schedule": Object {
    "data": Object {},
  },
  "shared": Object {
    "filter": Object {},
    "loading": Object {},
  },
  "spotify": Object {
    "devices": Object {
      "load": Object {
        "error": null,
        "isFailure": false,
        "isPending": false,
        "isSuccess": false,
      },
      "value": Array [],
    },
    "player": Object {
      "load": Object {
        "error": null,
        "isFailure": false,
        "isPending": false,
        "isSuccess": false,
      },
      "play": Object {
        "load": Object {
          "error": null,
          "isFailure": false,
          "isPending": false,
          "isSuccess": false,
        },
      },
    },
    "searchResults": Object {
      "load": Object {
        "error": null,
        "isFailure": false,
        "isPending": false,
        "isSuccess": false,
      },
      "value": Object {},
    },
    "simplifiedAlbums": Object {},
    "spotifyAccessToken": null,
    "tracks": Object {},
  },
}
`;
